tracee-ebpf/tracee/events_pipeline.go

```diff
@@ -222,26 +245,12 @@ func (t *Tracee) prepareEventForPrint(done <-chan struct{}, in <-chan RawEvent)
 				errc <- err
 				continue
 			}
+			t.stats.eventCounter.Increment()
```

Comment:
why increment in "prepare event for print"? "eventsCounter should report processed events

Reply:
Not exactly.
Until now, the count was incremented in printEvent, which was even after this stage.
I think that we want to keep it that way, because sometimes there are events that weren't chosen by the user, but we still process them (for example, to capture executed files). In fact, any essential event that was not chosen by the user shuoldn't be accounted.